.github/workflows/schema-CDN.yml

```diff
@@ -16,18 +16,21 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v3
+        uses: aws-actions/configure-aws-credentials@v4
         with:
           role-to-assume: arn:aws:iam::155736826933:role/GITHUB_ACTION
           role-session-name: githubactions
           aws-region: eu-north-1
 
       - name: Sync Files to S3
         run: |
-          aws s3 sync . s3://schema-cdn-service-bucket --exclude ".git/*" --delete
+          aws s3 sync . s3://schema-cdn-service-bucket \
+            --exclude "*" \
+            --include "*.schema.json" \
```

Comment:
Could we update this to specify that only `schema.json` files inside the `schemas` folder should be included? I think the pattern would look something like `schemas/**/schema.json`